pkg/build/gobuild.go

```diff
@@ -82,26 +85,54 @@ func NewGo(options ...Option) (Interface, error) {
 	return gbo.Open()
 }
 
+// goList is a wrapper for the `go list` command to retrieve information about a package.
+func (g *gobuild) goList(importPath, projection string) (string, error) {
```

Comment:
What does `projection` here mean? Is that a known term for `go list`?

Reply:
changed to format, since it is the name of the flag on go list.